Zappy_IA_Report/DTC/Code/data.py

```
@@ -0,0 +1,62 @@
+    df = load_data()
+    try:
+        if len(df) < 10:
+            e = "[ML] Not enough data to train model"
+        else:
+            X = df[FEATURES]
+            y = df[["action"]]
+            model = DecisionTreeClassifier()
+            model.fit(X, y)
+            joblib.dump(model, MODEL_FILE)
+            print("[ML] Trained model and saved to", MODEL_FILE)
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)